tests/README.md

```diff
@@ -155,3 +155,17 @@ When writing new tests, please implement all functions in `.cpp` or `.c` files,
 ## Writing fuzz tests
 
 [TODO]
+
+## Runing Blake2b Test
```

Comment:
I don't think we need an extra header for blake2b test, they are just another hash test which we already have a header for. You can maybe add some of this documentation to hash test.

Reply:
Sure. Will fix this.